src/test/java/api/clients/CardTestRestClient.java

```diff
@@ -40,12 +44,24 @@ public ValidatableResponse deleteCardIfExist(String cardId) {
                 .statusCode(anyOf(is(HTTP_OK), is(HTTP_NOT_FOUND)));
     }
 
-    public Response getCardInfoByCardId(String cardId) {
+    public Response getCardId(String cardId) {
         return given()
                 .spec(requestSpec)
                 .get("/1/cards/{id}", cardId);
     }
 
+    public CardDataResponse getCardById(String cardId) {
+        return given()
+                .spec(requestSpec)
+                .get("/1/cards/{id}", cardId)
+                .then()
+                .extract().as(CardDataResponse.class);
+    }
+
+    public static Map<String, String> constructDefaultCardKeyValueWithDesc() {
+        return Map.of("name", NAME_VALUE, "desc", DESC_VALUE);
+    }
```

Comment:
Rewrite the method so that it receives the name, description parameters when called